generators/server/templates/app/src/test/java/repository/SchemaValidationTest.java

```diff
@@ -0,0 +1,21 @@
+package <%= packageName %>;
+
+import <%= packageName %>.common.AbstractIntegrationTest;
+import org.junit.jupiter.api.Test;
+import org.springframework.test.context.TestPropertySource;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import static org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase.Replace.NONE;
+
+@TestPropertySource(properties = {"spring.jpa.hibernate.ddl-auto=validate"})
+@ExtendWith(SpringExtension.class)
+@DataJpaTest
+@AutoConfigureTestDatabase(replace = NONE)
+class SchemaValidationTest extends AbstractIntegrationTest {
```

Comment:
1. This class is extending AbstractIntegrationTest which is annotated with @SpringBootTest. That makes this is an integration test. So, we don't need to add @DataJpaTest.
2. Adding @ExtendWith(SpringExtension.class) is not required because @SpringBootTest, @DataJpaTest etc are already meta-annotated with @ExtendWith(SpringExtension.class)
3. Again, as it is an integration test we don't need @AutoConfigureTestDatabase(replace = NONE)

The existing ApplicationIntegrationTest is for validating whether the application can be bootstrapped correctly or not and also doing the schema validation testing. So, this PR is redundant.